tests/api/entities/resolver/resolve_and_update.test.js

```diff
@@ -181,6 +181,89 @@ describe('entities:resolver:update-resolved', () => {
     batchId.should.above(startTime)
     batchId.should.below(Date.now())
   })
+
+  it('should not update if entry date is as precise as entity date', async () => {
+    const year = '2020'
+    const { uri, isbn } = await createEditionWithIsbn({ publicationDate: year })
+    const entry = {
+      edition: {
+        isbn,
+        claims: { 'wdt:P577': year }
+      }
+    }
+    const { entities: preResolvedEntities } = await getByUris(uri)
+    const preResolvedEntityVersion = Object.values(preResolvedEntities)[0].version
+
+    await resolveAndUpdate(entry)
+    await wait(10)
+    const { entities: postResolvedEntities } = await getByUris(uri)
+    const postResolvedEntityVersion = Object.values(postResolvedEntities)[0].version
+    postResolvedEntityVersion.should.equal(preResolvedEntityVersion)
+  })
+
+  it('should not update if entry date disagree with entity date', async () => {
+    const entryDate = '2020-01-01'
+    const entityDate = '2021'
+    const { uri, isbn } = await createEditionWithIsbn({ publicationDate: entityDate })
+    const entry = {
+      edition: {
+        isbn,
+        claims: { 'wdt:P577': entryDate }
+      }
+    }
+    const { entities: preResolvedEntities } = await getByUris(uri)
+    const preResolvedEntityVersion = Object.values(preResolvedEntities)[0].version
+
+    await resolveAndUpdate(entry)
+    await wait(10)
+    const { entities: postResolvedEntities } = await getByUris(uri)
+    const postResolvedEntityVersion = Object.values(postResolvedEntities)[0].version
```

Comment:
[detail] we have a `getByUri` function to make this classic pattern less verbose
```suggestion
    const { version: postResolvedEntityVersion } = await getByUri(uri)
```